app/src/main/kotlin/de/digitalService/useID/ui/screens/setup/SetupPersonalPinIntro.kt

```diff
@@ -0,0 +1,131 @@
+package de.digitalService.useID.ui.screens.setup
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Info
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.ViewModel
+import com.ramcosta.composedestinations.annotation.Destination
+import dagger.hilt.android.lifecycle.HiltViewModel
+import de.digitalService.useID.R
+import de.digitalService.useID.ui.components.*
+import de.digitalService.useID.ui.coordinators.SetupCoordinator
+import de.digitalService.useID.ui.theme.UseIdTheme
+import javax.inject.Inject
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Destination
+@Composable
+fun SetupPersonalPinIntro(viewModel: SetupPersonalPinIntroViewModelInterface = hiltViewModel<SetupPersonalPinIntroViewModel>()) {
+    ScreenWithTopBar(
+        navigationButton = NavigationButton(
+            icon = NavigationIcon.Back,
+            onClick = viewModel::onBackButtonClicked
+        )
+    ) { topPadding ->
+        StandardButtonScreen(
+            primaryButton = BundButtonConfig(
+                title = stringResource(id = R.string.firstTimeUser_personalPINIntro_continue),
+                action = viewModel::onSetPin
+            ),
+            modifier = Modifier.padding(top = topPadding)
+        ) {
+            Column(
+                modifier = Modifier
+                    .padding(horizontal = UseIdTheme.spaces.m)
+                    .verticalScroll(rememberScrollState())
+            ) {
+                Text(
+                    stringResource(id = R.string.firstTimeUser_personalPINIntro_title),
+                    style = UseIdTheme.typography.headingXl
+                )
+
+                Spacer(modifier = Modifier.height(UseIdTheme.spaces.m))
+
+                Card(
+                    colors = CardDefaults.cardColors(containerColor = UseIdTheme.colors.blue200),
```

Comment:
Could you please check on the alignment of the elements of the card again?

Reply:
I know the card appearance doesn't match the current design. I decided to not change it (only adapt font, spacing etc. of new design system) because there was [this](https://digitalservicebund.atlassian.net/jira/software/projects/USEID/boards/3/backlog?label=Android%2CApp&selectedIssue=USEID-612) ticket which will require a rework of the card anyways 🤔

Reply:
Couldn't (quickly) figure out a way to implement the design from figma. I fixed the alignment of the icon with the header text and asked Tom if we can change the design so its easier to implement (answer still pending). But for now, I think we can leave it like that. (the card might change anyways, see ticket from comment above)

Reply:
Tom got back to me and said this should be implemented as it is defined in the design. So I will have to work on that again, but maybe on another PR/ ticket?

Reply:
No, then let's address that in this PR please. If you can't find I way, just tell me and I will take a look myself. No worries.

Reply:
Alright :) I fixed it with a rather simple solution but maybe not the cleanest one 🤔